impl.go

```diff
@@ -447,7 +459,19 @@ to prevent shell globbing.
 		}
 	}
 
-	fns, err := funcs(iface, *flagSrcDir, EmitComments(*flagComments))
+	if *flagRecvPkg == "" {
+		//  "   s *Struct   " , receiver: Struct
```

Comment:
I think that something along these lines simpler and clearer (but keeping your original comments, which I elided):

```go
recvs := strings.Fields(recv)
recv := recvs[len(recvs)-1] // note that this correctly handles "s *Struct" and "*Struct"
recv = strings.TrimPrefix(recv, "*")
```

Reply:
Good idea.